### PR TITLE
Creating the QPAS helper library 

### PR DESCRIPTION
**Pull request by Avilir**
This PR is starting to build the QPAS helper library, which contain performance tuned function.

Adding the qpas run_command() function

this is the firs function which used in almost all of the QPAS tests (performance & scale).

### CODE REVIEW

**Review by ypersky1980 — APPROVED** [review]

**Review by wusui — COMMENTED** [review]
If out_format is a list, the last element is removed.  If the out_format is a string, does the last element not get removed?

**Comment by Avilir** [comment]
> If out_format is a list, the last element is removed. If the out_format is a string, does the last element not get removed?

The default output is string with NL as separator between lines, when you split this 'long string' to lines (using NL as separator) you are getting an extra empty element in the end which remove from the list.
it does not exist as element in the original string

**Review by wusui — DISMISSED** [review]
Okay. Looks good

**Comment by ramkiperiy** [comment]
@Avilir the code looks good, since there is no existing test to verify this function what is the plan for verification ?
Any local run logs available ? if not then unit-testing code need to be written.

**Comment by Avilir** [comment]
> @Avilir the code looks good, since there is no existing test to verify this function what is the plan for verification ?
> Any local run logs available ? if not then unit-testing code need to be written.

since we do not need cluster to test this function, i test it locally and not via PR validation.

llocal test output :

```
======================================================================================= test session starts =======================================================================================
platform linux -- Python 3.9.1, pytest-5.3.5, py-1.9.0, pluggy-0.13.1
rootdir: /home/avili/Testing/OCS-CI, inifile: pytest.ini
plugins: metadata-1.10.0, marker-bugzilla-0.9.4, ordering-0.6, html-2.1.1, logger-0.5.1, reportportal-1.10.3
collected 1 item                                                                                                                                                                                  

tests/avi_test.py::test_run 
----------------------------------------------------------------------------------------- live log setup ------------------------------------------------------------------------------------------
08:59:44 - MainThread - tests.conftest - INFO - All logs located at /tmp/ocs-ci-logs-1608706781
08:59:44 - MainThread - ocs_ci.utility.utils - INFO - Executing command: /home/avili/Testing/OCS-CI/bin/oc version --client
08:59:44 - MainThread - ocs_ci.utility.utils - INFO - OpenShift Client version: Client Version: 4.7.0-0.nightly-2020-12-21-131655

08:59:44 - MainThread - tests.conftest - INFO - Skipping version reporting for development mode.
08:59:44 - MainThread - ocs_ci.utility.utils - INFO - testrun_name: OCS4-7-Downstream-OCP4-7-VSPHERE-UPI-1AZ-RHCOS-3M-3W-
08:59:44 - MainThread - tests.conftest - INFO - Skipping health checks for development mode
------------------------------------------------------------------------------------------ live log call ------------------------------------------------------------------------------------------
08:59:44 - MainThread - ocs_ci.helpers.performance_lib - INFO - Going to format output as list
08:59:44 - MainThread - ocs_ci.helpers.performance_lib - INFO - Going to run ls -l /etc/hosts with timeout of 600
08:59:44 - MainThread - tests.avi_test - INFO - Output as a list : ['-rw-r--r-- 1 root root 178 Nov 12 09:48 /etc/hosts']
08:59:44 - MainThread - ocs_ci.helpers.performance_lib - INFO - Going to format output as string
08:59:44 - MainThread - ocs_ci.helpers.performance_lib - INFO - Going to run ls -l /etc/hosts with timeout of 600
08:59:44 - MainThread - tests.avi_test - INFO - Output as a string : -rw-r--r-- 1 root root 178 Nov 12 09:48 /etc/hosts

PASSED                                                                                                                                                                                      [100%]

======================================================================================== warnings summary =========================================================================================
.venv/lib64/python3.9/site-packages/_pytest/config/__init__.py:891
  /home/avili/Testing/OCS-CI/.venv/lib64/python3.9/site-packages/_pytest/config/__init__.py:891: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: pytest_reportportal
    self._mark_plugins_for_rewrite(hook)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
================================================================================== 1 passed, 1 warning in 0.15s ===================================================================================

```

File: ocs_ci/helpers/performance_lib.py [hunk 1]
```diff
@@ -37,3 +38,51 @@ def write_fio_on_pod(pod_obj, file_size):
...
+    if isinstance(cmd, str):
+        command = cmd.split()
+    elif isinstance(cmd, list):
+        command = cmd
+    else:
+        return "Error in command"
+
+    for key in ["stdout", "stderr", "stdin"]:
+        kwargs[key] = subprocess.PIPE
+
```
Comment (ypersky1980) [inline ocs_ci/helpers/performance_lib.py:68]: @Avilir What will happen if stdout/stderr/stdin values will be already passed to this function in kwargs? Wouldn't it be overwritten here by this code?

**Review by ypersky1980 — APPROVED** [review]

File: ocs_ci/helpers/performance_lib.py (hunk 1 above)
Comment (Avilir) [reply ocs_ci/helpers/performance_lib.py:68]: that is correct, but the purpose of this function is to capture and return all this data.
if (for instance) you want to send it to file, you need to use the subprocess in a different way.

**Review by ebenahar — APPROVED** [review]

**Pull request merged**